tests/test_utils/test_utils.py

```diff
@@ -51,6 +51,36 @@ def test_compress(self):
         self.assertTrue(os.path.exists(uncompressed_dir + '/' + test_file_name))
 
 
+    def test_deb_components_valid(self):
+        '''
+            Validate sources.list url can be parsed properly by checking
+            url, release and areas values match between origin and parsed.
+        '''
+        url = 'http://some.deb.mirror/debian/'
```

Comment:
Totally confused me.

Reply:
What is confusing, exactly?

Reply:
I never saw an answer as to why you initialize the url variable with a bogus value.

Reply:
This is a unit test. This particular test validates parsing invalid url.

Reply:
Oh....maybe a value like

url = 'http://this.is.supposed.to.fail/DeBiAn/'

would be better?   Or a comment # this is supposed to fail

If you disagree, just leave it and complete the merge.

Reply:
Oh wait, actually, this is just an arbitrary url that will be parsed by utils function. And since function parses a string and doesn't do actual request calls, it doesn't matter what "url" is. That is why I called it "some.deb.mirror". Now I remember... :)